exporter/googlecloudexporter/config.go

```diff
@@ -86,6 +87,10 @@ func createDefaultGCPConfig() *gcp.Config {
 	config.UserAgent = defaultUserAgent
 	config.MetricConfig.Prefix = defaultMetricPrefix
 	config.LogConfig.DefaultLogName, _ = os.Hostname()
+
+	// Overwrites the default resource filter to match all resource attributes
+	defaultResourceFilter := collector.ResourceFilter{Prefix: ""}
+	config.MetricConfig.ResourceFilters = []collector.ResourceFilter{defaultResourceFilter}
```

Comment:
Should we document that we do this somewhere?

Reply:
I've updated the PR to include documentation that calls out this behavior.